src/Background.cpp

```diff
@@ -0,0 +1,15 @@
+#include "Background.h"
+#include <SFML/Graphics.hpp>
```

Comment:
this include is unnecessary . You are already including Background.h. Remember, that all that include does is literally copy the content of Backgroung.h, and put it here. So since .h file aready had include sfml graphics, you don't need to repeat it again here :)